lib/alias.ps1

```diff
@@ -0,0 +1,83 @@
+. "$PSScriptRoot\..\lib\install.ps1" # shim related
```

Comment:
Don't load library insides another library, move it to `scoop-alias.ps1` and add `install.ps1` to test file.